sdk-actors/src/test/java/io/dapr/actors/it/services/springboot/DaprController.java

```diff
@@ -0,0 +1,83 @@
+package io.dapr.actors.it.services.springboot;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dapr.actors.runtime.ActorManagerTest;
+import io.dapr.actors.runtime.ActorRuntime;
+import io.dapr.actors.runtime.ActorRuntimeConfigIT;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Map;
+import java.util.TimeZone;
+
+/**
+ * SpringBoot Controller to handle callback APIs for Dapr.
+ */
+@RestController
+public class DaprController {
+
+  @Autowired
+  private ObjectMapper objectMapper;
+
+  private String topics;
+
+  @GetMapping("/")
+  public String index() {
+    return "Greetings from Dapr!";
+  }
+
+  @GetMapping("/dapr/config")
+  public String daprConfig() throws Exception {
```

Comment:
Remove the commented code and also please add a comment why this configuration is being provided (e.g to intentionally decrease the deactivation time because the test tries to validate deactivation, etc).